etl/definitions.py

```diff
@@ -1,46 +1,10 @@
-import json
-
-from dagster import Definitions, EnvVar, FilesystemIOManager, load_assets_from_modules
-
-from etl.models import OpenAiResourceParams, OpenAiSettings
-from etl.models.types import EnrichmentType, RecordType
-from etl.resources import (
-    OpenAiEmbeddingModelResource,
-    OpenAiGenerativeModelResource,
-    WikipediaReaderResource,
-)
+from dagster import Definitions, load_assets_from_modules
 
 from . import assets
-from .assets import embedding_job
+from .jobs import embedding_job
 
 all_assets = load_assets_from_modules([assets])
```

Comment:
Nit: you could probably inline these variables in the Definitions construction since you're not exporting them.

Reply:
Yes, these variables are in Definitions now.